### ISSUE TITLE
Promise based TextField error message is not read by firefox + NVDA

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#4295]
### Bug Report

- __Package version(s)__: https://codepen.io/anon/pen/LdRVZQ
- __Browser and OS versions__: Firefox 58.0.2 on windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: visualstudio.com

#### Describe the issue:

When error message is delayed and user moves to next item, error message is not announced by NVDA.

#### Actual behavior:

Error message is not announced even though in live region.

#### Expected behavior:

Error message should be announced.

### If applicable, please provide a codepen repro:

https://codepen.io/anon/pen/LdRVZQ
Steps: 
-> Open above link in Firefox and turn NVDA ON.
-> Type "123" in first text field and immediately press tab
-> Wait till error message appears
This error message should have been announced but NVDA doesn't announce anything.
